app/templates/settings/integrations/index.hbs

```diff
@@ -1,13 +1,13 @@
 <section class="gh-canvas">
     <header class="gh-canvas-header">
-        <h2 class="gh-canvas-title" data-test-screen-title>Apps</h2>
+        <h2 class="gh-canvas-title" data-test-screen-title>Integrations</h2>
     </header>
 
     <section class="apps-grid-container">
-        <span class="apps-grid-title">Available integrations</span>
+        <span class="apps-grid-title">Built-in integrations</span>
```

Comment:
Ah okay :grin: shall i merge?

Reply:
Need to check if we're merging before everything else is ready because it will have an impact on docs - all the existing "Apps" docs have screenshots and descriptions that will need updating.

Reply:
no rush for now i guess